system/jlib/jlog.cpp

```diff
@@ -3112,3 +3122,63 @@ IRemoteLogAccess *queryRemoteLogAccessor()
     );
 }
 
+bool doTrace(TraceFlags featureFlag, TraceFlags level)
+{
+    if ((threadTraceFlags & TraceFlags::LevelMask) < level)
+        return false;
+    return (threadTraceFlags & featureFlag) == featureFlag;
+}
+
+void setTraceFlag(TraceFlags flag)
```

Comment:
It may be that setXX is not needed any more if the load functions are always used. The distinction is it's used to turn on a single bit rather than overwrite the whole set.

I hate naming!